guide/src/main/scala/io/udash/web/guide/views/ext/UserActivityExtView.scala

```diff
@@ -0,0 +1,167 @@
+
+  override def handleState(state: UserActivityExtState.type): Unit = {
+    rpc.clearCalls()
+  }
+
+  def reload(): Unit = {
+    rpc.calls.onComplete {
+      case Success(calls) => model.set(calls)
+      case Failure(t) => logger.error(t.getMessage)
+    }
+  }
+}
+
+
+case object UserActivityExtViewPresenter extends ViewPresenter[UserActivityExtState.type] {
+
+  import io.udash.web.guide.Context._
+
+  override def create(): (View, Presenter[UserActivityExtState.type]) = {
+    val model = SeqProperty[Call]
+    val presenter = new UserActivityExtPresenter(model)
+    (new UserActivityExtView(model, presenter), presenter)
+  }
+}
+
+class UserActivityExtView(model: SeqProperty[Call], presenter: UserActivityExtPresenter) extends View {
+
+  import io.udash.web.guide.Context._
+
+  import JsDom.all._
+  import scalacss.ScalatagsCss._
+
+  override def getTemplate: dom.Element = div(
+    h1("Udash user activity monitoring"),
+    p(
+      """When it comes to website tracking there are a plethora of metrics at our disposal.
+        |If we’re talking user engagement, we might look at the bounce rate, average time on site, or average page views per visit.
+        |For organic search, we might look at the number of organic visits, top organic keywords, and others.
+        |Then there’s content — the king of all data. Which web pages are the most popular?
+        |What are the most common navigation flows? Which features are most commonly used?
+        |Udash user activity extenstions enable you to gather the data you need to provide the best user experience for your website.""".stripMargin
+    ),
+    h2("Browser navigation"),
+    p("To enable browser navigation tracking, simply mixin UrlLogging into your frontend application. ",
+      "The ", i("log(url, referrer)"), " method will be called whenever the user changes app state."
+    ),
+    p("You can see this mechanism in action here in the guide. We've already provided the implementation: "),
+    CodeBlock(
+      s"""val application = new Application[RoutingState](routingRegistry,
+          |viewPresenterRegistry, RootState) with UrlLogging[RoutingState] {
+          |  override protected def log(url: String, referrer: Option[String]): Unit =
+          |    UrlLoggingDemo.log(url, referrer)
+          |}""".stripMargin
+    )(GuideStyles),
+    CodeBlock(
+      s"""object UrlLoggingDemo {
+          |  import io.udash.web.guide.Context._
+          |
+          |  val enabled = Property(false)
+          |  val history = SeqProperty[(String, Option[String])](ListBuffer.empty)
+          |  enabled.listen(b => if(!b) history.set(ListBuffer.empty))
+          |
+          |  def log(url: String, referrer: Option[String]): Unit =
+          |    if(enabled.get) history.append((url, referrer))
+          |}""".stripMargin
+    )(GuideStyles),
+    p("to see it in action just enable logging below, switch to another chapter and come back here."), br,
+    span(GuideStyles.frame)(
```

Comment:
Sorry, I didn't notice this earlier, but could you separate this demo component into `io.udash.web.guide.views.ext.demo` package? Take a look at `DynamicRemoteTranslationsDemo` for example.